habitipy/cli.py

```diff
@@ -747,6 +796,11 @@ def main(self, *args, **kwargs):
 class Habits(TasksPrint):  # pylint: disable=missing-docstring
     DESCRIPTION = _("List, up and down habit tasks")  # noqa: Q000
     domain = 'habits'
+    tag = cli.SwitchAttr(
```

Comment:
Can you please try to move this to its base class (`TagsHelper`)?

Reply:
Also, please use a repeatable switch. [See plumbum docs on this](https://plumbum.readthedocs.io/en/latest/cli.html#repeatable-switches).